packages/client/src/generation/TSClient/Model.ts

```diff
@@ -570,3 +570,5 @@ ${f.name}<T extends ${getFieldArgName(f, !this.dmmf.typeMap[fieldTypeName])} = {
 }`
   }
 }
+
+type t = Buffer
```

Comment:
What does this do?
Looks like it's not needed to me 👀

Reply:
Yep, indeed. Thanks!